filament/backend/include/backend/DriverEnums.h

```diff
@@ -523,6 +535,18 @@ enum class TextureFormat : uint16_t {
     DXT1_RGB, DXT1_RGBA, DXT3_RGBA, DXT5_RGBA,
     DXT1_SRGB, DXT1_SRGBA, DXT3_SRGBA, DXT5_SRGBA,
 
+    // RGTC formats available with a GLES extension
+    RED_RGTC1,              // BC4 unsigned
+    SIGNED_RED_RGTC1,       // BC4 signed
+    RED_GREEN_RGTC2,        // BC5 unsigned
+    SIGNED_RED_GREEN_RGTC2, // BC5 signed
+
+    // BPTC formats available with a GLES extension
+    RGB_BPTC_SIGNED_FLOAT,  // BC6H signed
+    RGB_BPTC_UNSIGNED_FLOAT,// BC6H unsigned
+    RGBA_BPTC_UNORM,        // BC7
+    SRGB_ALPHA_BPTC_UNORM,  // BC7 sRGB
+
```

Comment:
- move at the end of the list
- update `Texture.java`

Reply:
Oops I've been excluding the `android` folder when searching the code because I didn't realize it contains code too (that's how I missed Texture.java). Checked the whole folder and only Texture.java needs a little update, done in 559ec06bb211a5c6eba48a09367b7364b05bf17e.

As for the enum order, see 1067be3b4f445feb370a1296e3f3d4a10306be8e

Reply:
These enums in java must match the C++ ones, so they also need to be moved at the end.

Reply:
The enum values are the same in DriverEnums.h and Texture.java.. What sort of mismatch do you mean?

Reply:
Well I mean that they must be at the end below `SRGB8_ALPHA8_ASTC_12x12`. The java code uses the order of the enums to convert them to the C++ enum. That's a bit fragile, but that's how it's done currently.
Do I make sense?

Reply:
🤦 I only now realized I forgot to update `TextureFormat`. Sorry, fixed in a52753b6f73ed3ef3b98ec384de0139ce41dcc64